src/components/App.tsx

```diff
@@ -262,6 +265,7 @@ import {
 const defaultDeviceTypeContext: DeviceType = {
   isMobile: false,
   isTouchScreen: false,
+  isFloatingMenu: false,
```

Comment:
this naming is confusing as its `isFloatingMenu` doesn't go with `deviceType` as its not a device type.

Can we keep it out of `deviceType` instead as its only sidebar specific and also rename to `canDeviceFitSidebar` or something?